README.md

```diff
@@ -52,6 +52,16 @@ production:
 
 Then run `db:prepare` in production to ensure the database is created and the schema is loaded.
 
+### Single database configuration
+
+Running Solid Cable in a separate database is recommended, but it's also possible to use one single database for both the app and the cable. Just follow these steps:
+
+1. Copy the contents of `db/cable_schema.rb` into a normal migration and delete `db/cable_schema.rb`
+2. Remove `connects_to` from `config/cable.yml`
+3. Migrate your database
+
+You won't have multiple databases, so `database.yml` doesn't need to have primary and cable database.
```

Comment:
Do we need this line? The generator doesnt touch this file so i think we might be able to get away with not having it.

Reply:
I implemented this based on what's in the Solid Queue README. I think it makes sense to keep it here to emphasize that the configuration in `database.yml` should be left out.